clients/vscode/package.json

```diff
@@ -270,6 +270,11 @@
                   "Automatic trigger when you stop typing",
                   "Manual trigger by pressing `Alt + \\`"
                 ]
+              },
+              "numRecentlyCommandsHistory": {
```

Comment:
```suggestion
              "numInlineChatHistory": {
```

Reply:
@wsxiaoys, for now, I don't use InlineChat to refer to this feature in the code. The most commonly used term is `ChatEdit`. https://github.com/TabbyML/tabby/blob/main/clients/tabby-agent/src/lsp/protocol.ts#L349.

Reply:
```suggestion
              "numChatEditCommandsHistory": {
```

Reply:
I'll take`chatEdit.history` :)